### DISCUSSION TITLE
Audi Q2 2017 MHI2_ER_AU276_P4160 cant find patch ? or firmware

### DISCUSSION THREAD

**Discussion by mickwillow** [Mr-MIBonk/M.I.B._More-Incredible-Bash#120] [General]
Hi all I have a Audi Q2 2018 in which i'm trying to update the sav nav maps. I have installed MIB and got the green menu up but cant find any patches or f/w  updates for my unit.
The MIB Helper show the latest official firmware is MHI2_ER_AU276_P5515 for my unit but can't find it when I click on the link . Could some please help me to what firm ware I could update to or a patch for my unit ? 
Many thanks Mick

**Comment by harman-f** [comment]
That version is not public, currently only available for new cars from factory. 

Update to latest available and patch with M.I.B

**Reply by mickwillow** [reply]
thanks for the reply, is my firm ware a old version?  could you send me a link for the right firm ware download as I can't find MHI2_ER_AU276_P5515 anywhere please, the only thing I've found as MHI2_ER_AU276_P5088_MU on mibsolution firmware page  thanks again

**Reply by harman-f** [reply]
5088 is the right one.

**Reply by mickwillow** [reply]
Great thanks I'll hit the download button , I'm i right in thinking that I just need  the corresponding  patch in the mib patch folder and can delete the rest?

**Reply by harman-f** [reply]
Just copy all over to SD

M.I.B will select the right one.

**Reply by mickwillow** [reply]
thanks for all your help Harman , its my first time dating firm ware on the car .

**Comment by mickwillow** [comment]
My Q2 has tech pack and virtual cockpit does that make any different to firm ware on the unit ?

**Reply by harman-f** [reply]
No, same FW for all, it is just different coding, which is already in place on unit and not touched by FW update or M.I.B.

**Comment by mickwillow** [comment]
Had a few hours spare tonight and managed to update firm ware and update the maps to 2023 on my Q2.  Big thanks to Harman
